package.json

```diff
@@ -27,7 +27,7 @@
     "fix": "npm run lint -- --fix",
     "flow": "flow",
     "lint": "eslint .",
-    "precommit": "lint-staged && flow && tsc",
+    "precommit": "yarn-deduplicate && lint-staged && flow && tsc",
```

Comment:
Please move this to another hook. The precommit hook should only verify things, not edit things. Maybe `prepare` or `fix`?